### PR TITLE
[16.0][MIG] pdf_helper

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

File: pdf_helper/tests/test_helper.py [hunk 1]
```diff
@@ -16,9 +16,7 @@ def read_test_file(filename, mode="r"):
         return thefile.read()
 
 
-# NOTE: this class could use a bare `unittest.TestCase` as base
-# but w/out TreeCase Odoo won't load these tests.
-class TestPDFHelperUtils(TreeCase):
+class TestPDFHelperUtils(TransactionCase):
```
Comment (simahawk) [inline pdf_helper/tests/test_helper.py]: can you please try using `TestCase` instead? https://github.com/odoo/odoo/blob/16.0/odoo/tests/case.py#L93

**Comment by simahawk** [comment]
@gbrito-erpopen cool, thanks! 1 last thing: please squash your last commits on the migration one.

**Comment by ghost** [comment]
@simahawk done

**Pull request merged**